darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -230,6 +257,45 @@ def _compute_loss(self, output, target):
             # last dimension of model output, for properly computing the loss.
             return self.criterion(output.squeeze(dim=-1), target)
 
+    def _setup_metrics(self):
+        if self.metrics_str:
+            self.metrics = []
+            task_module = torchmetrics.functional
```

Comment:
couldn't we directly do something like `import torchmetrics.functional as func_torch_metrics` at the top of the file? No need to redeclare it here, unless I'm missing something.